maths/fibonacci.py

```diff
@@ -230,6 +235,79 @@ def fib_binet(n: int) -> list[int]:
     return [round(phi**i / sqrt_5) for i in range(n + 1)]
 
 
+def matrix_mult_np(a, b):
+    """
+    Multiplies two matrices using numpy's dot product.
+
+    Args:
+        a: First matrix as a numpy array
+        b: Second matrix as a numpy array
+
+    Returns:
+        The product of matrices a and b.
+    """
+    return np.dot(a, b)
```

Comment:
```suggestion
```
This function isn't used anymore because we're using `numpy`'s matrix multiplication function, so we can just remove this function entirely.